administrator/modules/mod_guidedtours/tmpl/default.php

```diff
@@ -42,7 +44,12 @@
     // We assume the url is the starting point
     $key = $uri->getVar('option') ?? Text::_('MOD_GUIDEDTOURS_GENERIC_TOUR');
 
-    if (!isset($allTours[$key])) :
+    if (empty($tour->alias))
+    {
+        $lang->load( "com_guidedtours_" . str_replace( "-", "_", $tour->alias ), JPATH_ADMINISTRATOR );
+    }
```

Comment:
```suggestion
    if (empty($tour->alias)) :
        $lang->load( "com_guidedtours_" . str_replace( "-", "_", $tour->alias ), JPATH_ADMINISTRATOR );
    endif;
```
Let's keep the original syntax and formatting to make the code style checker happy.